code/script.js

```diff
@@ -1,47 +1,228 @@
 // All the DOM selectors stored as short variables
-const chat = document.getElementById('chat')
+const chat = document.getElementById("chat");
+const nameInput = document.getElementById("name-input");
+const submitBtn = document.getElementById("btn");
+const inputWrapper = document.getElementById("input-wrapper");
 
+//[updated] 'name' is reserved as global variable? instead 'name' -> 'username'
+let username;
```

Comment:
Nice to declare your variable username as a global variable since it's being used in different functions.